avocado/core/job.py

```diff
@@ -422,8 +422,7 @@ def _run(self):
         self._setup_job_results()
         self.__start_job_logging()
 
-        if (getattr(self.args, 'remote_hostname', False) and
-           getattr(self.args, 'remote_no_copy', False)):
+        if getattr(self.args, 'remote_execution', False):
```

Comment:
We can move such test to the loader itself, the same way I did in this PR for the multiplex.